tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -395,12 +396,17 @@ Set PVC Value In RHODS Dashboard
 
 Restore PVC Value To Default Size
     [Documentation]    Set the PVC value to default
-    ...    valie i.e., 20Gi
+    ...    value i.e., 20Gi
     Menu.Navigate To Page    Settings    Cluster settings
     Wait Until Page Contains Element  xpath://input[@id="pvc-size-input"]  timeout=30
     Click Button    Restore Default
-    Wait Until Keyword Succeeds    30    1
+    Run Keywords
+    ...    Wait Until Keyword Succeeds    30    1
     ...    Wait Until Page Contains    Cluster settings updated successfully.
+    ...    AND
+    ...    Sleep    20s    msg=NOTE: This change will cause juypterhub to restart. It will take 30 seconds before juypterhub will be available. #robocop:disable
```

Comment:
would it be useuful to have a dedicated kw for `waiting until cluster settings have been updated`? Will these lines be likely used again for other TCs?

Reply:
For now we don't have exactly like this. We have notebook culler which change the setting but i don't think it needs this much time. I would propose to keep this here as it is if in future we get something which is related we can write a common keyword